src/app/containers/CpsRecommendations/utils.js

```diff
@@ -0,0 +1,17 @@
+// Transfomrs a list of recommendation items to a story promo format
+const transformToStoryPromoItems = items =>
```

Comment:
The title and function are perfectly readable here. 🙂 Is this comment necessary?